x-pack/plugins/siem/public/resolver/view/graph_controls.tsx

```diff
@@ -160,8 +161,8 @@ export const GraphControls = styled(GraphControlsComponent)`
   position: absolute;
   top: 5px;
   right: 5px;
-  background-color: #d4d4d4;
-  color: #333333;
+  background-color: ${() => useResolverTheme().colorMap.graphControlsBackground};
+  color: ${() => useResolverTheme().colorMap.graphControls};
```

Comment:
i don't have too much experience with styled components. Seems like you're calling a react hook here. Is that valid?

Reply:
Not valid in react as it can't hook into any stateful logic available in the react components. Had this here initially when my hook was just a function returning that data. Gonna move it out from here and a couple other places and pass it through the resolver component before I move this PR out of draft